packages/expo-router/src/views/Navigator.tsx

```diff
@@ -156,14 +158,16 @@ function NavigatorSlot() {
   return descriptors[state.routes[state.index].key]?.render() ?? null;
 }
 
+const SlotNavigatorWrapper = isEdgeToEdge() && Platform.OS === 'android' ? View : SafeAreaView;
```

Comment:
```suggestion
const SlotNavigatorWrapper = process.env.EXPO_OS === 'android' && isEdgeToEdge() ? View : SafeAreaView;
```

More sane minification in babel

Reply:
Also probably makes sense to use a `Fragment` instead of `View`